src/main/java/org/opentripplanner/graph_builder/issues/AreaTooComplicated.java

```diff
@@ -4,13 +4,15 @@
 
 public class AreaTooComplicated implements DataImportIssue {
 
-  public static final String FMT = "Area %s is too complicated (%s > %s )";
+  public static final String FMT = "Area %s is too complicated (%s > %s)";
+  public static final String HTMLFMT =
+    "Area <a href='http://www.openstreetmap.org/%s'>'%s'</a> is too complicated (%s > %s)";
```

Comment:
Each OSM element has a method to generate an OSM link: https://github.com/opentripplanner/OpenTripPlanner/blob/3f2f6d6f0ccf01d7bd27967cb4639e6242f237c4/src/main/java/org/opentripplanner/openstreetmap/model/OSMNode.java#L76

However, you can change that in your follow-up PR which also converts the records.